documentation/providers.md

```diff
@@ -142,9 +142,10 @@ code to support this provider, we'd be glad to help in any way.
 * [Alibaba Cloud DNS](https://github.com/StackExchange/dnscontrol/issues/420)(#420)
 * [Constellix (DNSMadeEasy)](https://github.com/StackExchange/dnscontrol/issues/842) (#842)
 * [CoreDNS](https://github.com/StackExchange/dnscontrol/issues/1284) (#1284)
-* [EnCirca](https://github.com/StackExchange/dnscontrol/issues/1048) (#1048)
 * [EU.ORG](https://github.com/StackExchange/dnscontrol/issues/1176) (#1176)
+* [EnCirca](https://github.com/StackExchange/dnscontrol/issues/1048) (#1048)
```

Comment:
Doesn’t “en” come before “eu?”

Reply:
Yes, of course! No idea why GoLang applied a different sorting here. Tested again and can't reproduce it. Removed the affected commit via a git rebase. Thanks for attention! 👏